File: design-patterns/JavaDesignPatterns/src/com/bigboxcode/designpattern/command/uielement/TableUi.java
package com.bigboxcode.designpattern.command.uielement;

public class TableUi implements UiCommand {
    @Override
    public void print() {
        System.out.println("Printing Table");
    }

    @Override
    public void remove() {
        System.out.println("Removing Table");
    }
}

File: design-patterns/php/command/uielement/TableUi.php
<?php
// TableUi.php

namespace BigBoxCode\DesignPattern\Command\UiElement;


class TableUi implements UiCommand {
    public function print(): void {
        echo "Printing Table\n";
    }

    public function remove(): void {
        echo "Removing Table\n";
    }
}